### ISSUE TITLE
h3 in card title requires a H2 above for it to not fail accessibility.

### ISSUE THREAD

**Issue by winley** [NHSLeadership/nhsblocks#239]
This should be a h2 as deault rather than h3, otherwise it always fails accesibility checks. 

<div class="wp-block-nhsblocks-card1 nhsuk-grid-column-width nhsuk-care-card nhsuk-care-card--type is-style-urgent"><div class="nhsuk-care-card__heading-container"><**h3** class="nhsuk-care-card__heading">....

**Comment by maheshmuralip** [comment]
Thanks @winley 
We will look into it.

**Comment by maheshmuralip** [comment]
Thanks @winley  v1.3.24 has accessibility improvements relating to this.
